app/scripts.babel/contentscript.js

```diff
@@ -84,10 +84,12 @@ function toggle_columns() {
 
 function hide_columns() {
   chrome.storage.sync.get({ 'columnsToHide': '' }, function(items) {
-    let column_collection = items.columnsToHide.split(',')
-      .map(function(i) { return i.trim(); });
-    let css = generate_column_hiding_css(column_collection);
-    window.v1_column_hiding_style_el = add_css_to_document(css);
+    if (typeof items.columnsToHide != 'undefined') {
```

Comment:
If my understanding of chrome's storage API is correct, the `{ 'columnsToHide': '' }` bit should ensure that `items.columnsToHide` is always defined inside the callback function (either as whatever was stored or as the fallback default of empty string), so I don't think the undefined check is needed here.